### PR TITLE
br: update description rawkv and faq

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update the description of br rawkv and faq
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -276,6 +280,8 @@ br backup raw --pd $PD_ADDR \
 - "hex"：将输入的字符串视作十六进制数字。这是默认的编码方式。
 - "escape"：对输入的字符串进行转义之后，再编码为二进制格式。
 
+如果需要通配全部数据，可以通过传两个空值 `--start="" --end=""`来实现。
```
Comment (kennytm) [inline br/use-br-command-line-tool.md:283]: using the verb "通配" here sounds strange...

Comment (IANTHEREAL) [reply br/use-br-command-line-tool.md:283]: @3pointer PTAL

File: br/backup-and-restore-faq.md [hunk 2]
```diff
@@ -177,4 +177,8 @@ BR v4.0.9 备份统计信息使 BR 消耗过多内存，为保证备份过程正
 
 + BR 在恢复数据时，会修改 PD 的一些全局配置。如果同时使用多个 BR 进程进行恢复，这些配置可能会被错误地覆写，导致集群状态异常。
 + 因为 BR 在恢复数据的时候会占用大量集群资源，事实上并行恢复能获得的速度提升也非常有限。
-+ 多个 BR 并行恢复的场景没有经过测试，无法保证成功。
\ No newline at end of file
++ 多个 BR 并行恢复的场景没有经过测试，无法保证成功。
+
+## BR 备份恢复后监控显示磁盘使用空间不一致
+
+这个情况多数是因为备份时集群的数据压缩比率，和恢复时的集群默认不一致导致的，只要恢复的 checksum 阶段顺利通过，可以忽略这个问题，不影响正常使用。
```
Comment (shichun-0415) [inline br/backup-and-restore-faq.md:184]:
```suggestion
这个情况多数是因为备份时集群的数据压缩比率和恢复时的默认值不一致导致的，只要恢复的 checksum 阶段顺利通过，可以忽略这个问题，不影响正常使用。
```

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -222,6 +222,10 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 示例备份的增量数据记录 `(LAST_BACKUP_TS, current PD timestamp]` 之间的数据变更，以及这段时间内的 DDL。在恢复的时候，BR 会先把所有 DDL 恢复，而后才会恢复数据。 
 
+>
+> 以下功能是实验特性，如果在使用过程中遇到问题，可以在 [AskTUG](https://asktug.com/) 社区中提问。
+>
+
```
Comment (shichun-0415) [inline br/use-br-command-line-tool.md:228]: ”以下功能“指”加密备份数据“？如果是，这里的描述可以删除，因为下面的”警告“已经体现了，可以把”如果在使用过程中遇到问题，可以在 [AskTUG](https://asktug.com/) 社区中提问“整合到“警告”。

**Comment by shichun-0415** [comment]
@3pointer Could you take a look at the comments? Thanks.

**Pull request closed without merging**